tox.ini

```diff
@@ -95,6 +96,8 @@ deps =
     aiobotocore: asynctest
     aiobotocore: moto>=1.0.1
     aiobotocore: flask
+    aiopg: aiopg>=0.12.0
```

Comment:
not sure, but can't you do something like:
```
aiopg012: aiopg>=0.12,<0.13
aiopg013: aiopg>=0.13,<0.14
```

and then:
```
{py34,py35,py36}-aiopg{012,013}
```

or something similar to let it work, so that we will execute your tests 2 times, one for 0.12 and one for 0.13.